### PR TITLE
Bug: Check project for missing translations

### PR DESCRIPTION
**Pull request by sariita05**
Bug: Check project for missing translations

Re #400

### CODE REVIEW

File: contents/en/projects/5.mystaffpilot.yaml [hunk 1]
```diff
@@ -1,40 +1,76 @@
...
-
+  -
+    component: 'contact'
+    title: 'Sounds interesting?'
+    description: 'You want to know more about the project or our implementation? No matter if you are a customer or an applicant, contact us! Come by for a coffee in our offices in Munich or Augsburg - we look forward to meeting you!'
+    button: 'Contact us now'
+    subject: 'Information about the project'
+    body: |
+      Hello anfema team,
+      I just had a look at your {{projectInfo}} project and ...
```
Comment (Somberland) [inline contents/en/projects/5.mystaffpilot.yaml:76]: wording missing? or the new default?

**Review by Somberland — CHANGES_REQUESTED** [review]

File: contents/en/projects/5.mystaffpilot.yaml [hunk 2]
```diff
@@ -1,40 +1,76 @@
...
-    component: 'project-text'
+    component: 'content-quote'
+    anchorId: 'msp-quote'
+    source: |
+      Iris Naudßus <br>
+      Mitglied der Geschäftsleitung STAFFPILOT GmbH
     content: |
-      <h3>Technology</h3>
-
+      <p>Wir sind mit einer sehr komplexen Aufgabe an die anfema herangetreten: Programmierung einer Plattform, die für unsere Agentur-Partner und Kunden einen Komplett-Service zur Abwicklung von Promotion-Jobs bietet. Die anfema hat die Sätze "geht nicht", "gibt es nicht" nicht in Ihrem Portfolio,  daher haben wir per dato eine unseren Wünschen entsprechende Plattform, die unseren Agenturpartnern und unseren Kunden diesen kompletten Service liefert! Kurzfristige Änderungen und  Erweiterungen werden gut überlegt umgesetzt! Wir freuen uns, die anfema an unserer Seite als langjährigen Dienstleister zu wissen.</p>
```
Comment (Somberland) [inline contents/en/projects/5.mystaffpilot.yaml:56]: we should translate those to english as well or what do you think?

Comment (sariita05) [reply contents/en/projects/5.mystaffpilot.yaml:56]: Yes we should, but I don't have translations for that yet.

File: contents/en/projects/5.mystaffpilot.yaml (hunk 1 above)
Comment (sariita05) [reply contents/en/projects/5.mystaffpilot.yaml:76]: It's the new default for now.

File: contents/en/projects/5.mystaffpilot.yaml (hunk 2 above)
Comment (Somberland) [reply contents/en/projects/5.mystaffpilot.yaml:56]: if we do not have an official translation of such quotes from clients we should do this ourselves.

> We approached anfema with a very complex task: Programming a platform that offers our agency partners and clients a complete service for handling promotion jobs. anfema does not have the phrases "doesn't work", "doesn't exist" in its portfolio. Therefore, we have a platform that meets our requirements as of today and provides our agency partners and customers with this complete service! Short-term changes and extensions are implemented with great consideration! We are happy to have anfema at our side as a longstanding service provider.

**Pull request merged**